src/Chapter13.Tests/Listing13.15.Tests.cs

```diff
@@ -0,0 +1,24 @@
+﻿
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter13.Listing13_15.Tests;
+
+[TestClass]
+public class ProgramTests
+{
+    [TestMethod]
+    public void MainTest()
+    {
+        const string expectedOutput = "ValidInput";
```

Comment:
```suggestion
        const string expectedOutput = ""<<ValidInput>>ValidInput"";
```
TestTools.Console should be able to handle both input and output: https://github.com/IntelliTect/TestTools.Console?tab=readme-ov-file#basic-usage

This would also all you to simplify the Expect call to just need the expected string and the `Action`

Reply:
I believe I implemented it how you were expecting, that's much simpler than what I was trying to do. Thank you!

Reply:
I updated the errata to include your contribution and give you credit and pushed the change to your branch if you would like to make sure it looks ok

Reply:
Perfect!